src/Symfony/Bundle/FrameworkBundle/Routing/DelegatingLoader.php

```diff
@@ -46,16 +48,23 @@ public function __construct(ControllerNameParser $parser, LoggerInterface $logge
     }
 
     /**
-     * Loads a resource.
-     *
-     * @param mixed  $resource A resource
-     * @param string $type     The resource type
-     *
-     * @return RouteCollection A RouteCollection instance
+     * {@inheritdoc}
      */
     public function load($resource, $type = null)
     {
-        $collection = parent::load($resource, $type);
+        if (false !== $this->loading) {
+            // This can happen if a fatal error occurs in parent::load()
```

Comment:
i don't get this. let me recap:
- the first time load is called parent::load throws an exception which is caught and loading is set back to false
- the second time load is called (to render exception page) parent::load throws the same error again of course, but this time the exception is not caught by php??? so loading is NOT set back to false
- the third time load is called (why third time???), this condition will throw an error straight away without calling parent:load